app/components/comments/CommentBubble.tsx

```diff
@@ -40,6 +55,41 @@ const CommentBubble = ({
   const [editMode, setEditMode] = useState(false);
   const [editedText, setEditedText] = useState(text);
 
+  const editEditor = useEditor({
```

Comment:
Let's define the common configuration in a separate file so it can be used in multiple places but still be overridden if needed.